src/knex-tables.d.ts

```diff
@@ -183,12 +183,13 @@ declare module "knex/types/tables" {
     middle_name?: string;
     last_name: string;
     status: RemovalStatus;
+    manually_resolved: boolean;
     created_at: Date;
     updated_at: Date;
   }
   type OnerepScanResultOptionalColumns = Extract<
     keyof OnerepScanResultRow,
-    "middle_name"
+    "manually_resolved" | "middle_name"
```

Comment:
Hm, I don't understand why this type would be `"manually_resolved" | "middle_name"`, would you mind elaborating on this?